fuzzers/aflplusplus_lto_cmplog_mopt2/Dockerfile

```diff
@@ -0,0 +1 @@
+builder.Dockerfile
```

Comment:
Sorry, I'm not sure what's happening here.

Reply:
ah that was a symlink so I can easily build the container during trying to get llvm 11 and gcc 9 running in the container. just removed it and removed also libfuzzer_valueprofile